Formula/varnish.rb

```diff
@@ -61,5 +61,18 @@ def install
 
   test do
     assert_match version.to_s, shell_output("#{sbin}/varnishd -V 2>&1")
+
+    # extract a subset of the varnishtest tests:
+    # - b*.vtc (basic functionality)
+    # - m*.vtc (VMOD modules, including loading), but skipping m00000.vtc which is known to fail
+    #   but is "nothing of concern" (see varnishcache/varnish-cache#3710)
+    # - u*.vtc (utilities and background processes)
+    system "tar", "-xzf", cached_download, "-C", testpath,
+        "--strip-components=4", "--exclude", "m00000.vtc",
+        "varnish-*/bin/varnishtest/tests/[bmu]*.vtc"
+    # -j: run the tests (using up to half the cores available)
+    # -q: only report test failures
+    # varnishtest will exit early if a test fails (use -k to continue and find all failures)
+    shell_output("#{bin}/varnishtest -j #{[Hardware::CPU.cores / 2, 1].max} -q #{testpath}/*.vtc")
```

Comment:
```suggestion
    system bin/"varnishtest, "-j", [Hardware::CPU.cores / 2, 1].max, "-q", *testpath.glob("*.vtc")
```
If we're just looking for an exit code of `0` then using `system` should suffice, I think.

Reply:
Check, I had issues with `system` before seemingly exiting too early and not failing the test, but I'll try this again.